src/main/java/frc/robot/imu/ImuSubsystem.java

```diff
@@ -17,16 +16,16 @@ public ImuSubsystem(Pigeon2 imu) {
 
   @Override
   protected void collectInputs() {
-    robotHeading = Rotation2d.fromDegrees(imu.getYaw().getValue());
+    robotHeading = imu.getYaw().getValue();
```

Comment:
Does this break anything else?

Reply:
We don't have hardly any higher level automation/logic written, so there's no existing code that uses the IMU data directly except for localization IIRC